### ISSUE TITLE
Image rotated on Kindle 5th gen running firmware 4.1.4

### ISSUE THREAD

**Issue by tac0tac0tac0** [leotaku/kojirou#32]
Im having the problem that the image of all chapters are rotated to landscape view, only showing half the page with no possibility to rotate the book to portrait or even scrolling down to the bottom half on my kindle 5th gen running firmware 4.1.4

**Comment by leotaku** [comment]
That's quite an interesting issue. Have you tried switching to landscape mode and maybe back? Does that do anything?

You could also try the `--left-to-right` flag. I'm not exactly sure why such an issue would be caused by right to left mode, but the setting seems to cause incompatibilities sometimes.

**Comment by leotaku** [comment]
Yeah, sorry. I can't really do much about this as I don't have a 5th Gen Kindle to test with.  Have you confirmed that Manga/Comic/Picture-Book type e-books from a different source can display correctly on your Kindle? If so you could send me such an e-book to me and that might help me with debugging.

**Comment by Dunnymeister** [comment]
I'm experiencing the same issue with a Kindle 4. I experience the same behaviour when opening a file generated by Kojirou in the [Desktop Kindle App](https://www.amazon.com.au/gp/help/customer/display.html?nodeId=GZSM7D8A85WKPYYD) on my Windows machine.

**Comment by leotaku** [comment]
@tac0tac0tac0 @Dunnymeister If you are still interested in this issue, could you please provide me with your exact Kindle model. Ideally also provide your serial number, which you can find in the settings menu. I am considering purchasing a used Kindle to properly debug this issue, however I'd rather not accidentally buy a model where the problem is not present.

Thank you for your help!
